package.json

```diff
@@ -11,11 +11,13 @@
     "docs:build": "pnpm run --filter=tutorialkit.dev build",
     "demo": "pnpm run --filter=demo.tutorialkit.dev dev",
     "demo:build": "pnpm run --filter=demo.tutorialkit.dev build",
+    "lint": "eslint \"{packages,docs}/**/*.{ts,tsx,js,json,jsonc,js,mjs,cjs}\"",
```

Comment:
Is eslint covering prettier as well?

Any reason we don't include all files and folders?

Reply:
Yes, our eslint plugin always included the eslint recommended rules and it also takes the prettier config into account.

> Any reason we don't include all files and folders?

I only did this to reduce the number of files to be checked and to be more specific. Other than that there's no particular reason. Cause those extensions are the only files that our plugin covers, but I think it would be fine to remove the extensions since the plugin contains the file extensions too and it would ignore other files.